wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
```

Comment:
Dunno what "embedded" means.  Like rails decks?

Does anyone actually need to do any of this stuff?  Dropping and reloading the schema seems to be a simple way to avoid schema issues in this case.  When is it important not to do that?